packages/main/src/RangeSlider.hbs

```diff
@@ -1,14 +1,61 @@
 {{>include "./SliderBase.hbs"}}
 
+<span id="{{_id}}-sliderDesc" class="ui5-hidden-text">{{_ariaLabelledByText}}</span>
```

Comment:
We had these line in both Slider's and RangeSlider's template, maybe it will be a bit better to move it to the SliderBase.hbs